pkg/queue/cluster_queue_impl.go

```diff
@@ -235,7 +247,11 @@ func (c *clusterQueueBase) Pending() int {
 }
 
 func (c *clusterQueueBase) PendingActive() int {
```

Comment:
Is this ever called without locking?

Reply:
No, it is called from two places:
- `cq.Pending()` - this is using read lock scoped to the cq
- `manager.reportPendingWorkloads()`, but all invocations of this function take manager lock, which is broader range than cq